compiler-rt/lib/profile/InstrProfilingPlatformWindows.c

```diff
@@ -74,7 +74,14 @@ ValueProfNode *__llvm_profile_end_vnodes(void) { return &VNodesEnd; }
 ValueProfNode *CurrentVNode = &VNodesStart + 1;
 ValueProfNode *EndVNode = &VNodesEnd;
 
+uint64_t BuildIdLen = 16;
+COMPILER_RT_WEAK extern uint8_t __buildid[16];
```

Comment:
Is this variable defined by lld?  Document the behavior.

Reply:
Yes, it was documented here: https://lld.llvm.org/windows_support.html#lld-flags

Reply:
Add a brief comment in the code for reference.

Reply:
Done.